core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -709,7 +709,7 @@ public enum CoreError implements ScalarDbError {
   CONSENSUS_COMMIT_CONFLICT_OCCURRED_WHEN_COMMITTING_STATE(
       Category.CONCURRENCY_ERROR,
       "0015",
-      "The committing state in the coordinator failed. The transaction has been aborted",
+      "A transaction conflict occurred when committing the state to the coordinator failed. The transaction has been aborted",
```

Comment:
This is not directly related to this change. But I included it since I think the message is something wrong and it's commit conflict thing.